src/script/entity/Conversation.ts

```diff
@@ -297,6 +298,10 @@ export class Conversation {
 
     this.call = ko.observable(null);
 
+    this.mls1To1ConversationState = this.mls1To1ConversationState = ko.observable<ONE_TO_ONE_CONVERSATION_STATE>(
```

Comment:
This looks weird 😅

Reply:
fixed this [15fee64bd78ce2c9432fdb7a9221307080fb1ff3](https://github.com/wireapp/wire-webapp/pull/15759/commits/15fee64bd78ce2c9432fdb7a9221307080fb1ff3)